sensing/pointcloud_preprocessor/src/blockage_diag/blockage_diag_nodelet.cpp

```diff
@@ -38,26 +39,41 @@ BlockageDiagComponent::BlockageDiagComponent(const rclcpp::NodeOptions & options
     lidar_model_ = static_cast<std::string>(declare_parameter("model", "Pandar40P"));
     blockage_count_threshold_ =
       static_cast<uint>(declare_parameter("blockage_count_threshold", 50));
-    buffering_frames_ = static_cast<uint>(declare_parameter("buffering_frames", 100));
-    buffering_interval_ = static_cast<uint>(declare_parameter("buffering_interval", 5));
+    blockage_buffering_frames_ =
+      static_cast<int>(declare_parameter("blockage_buffering_frames", 100));
+    blockage_buffering_interval_ =
+      static_cast<int>(declare_parameter("blockage_buffering_interval", 5));
+    dust_ratio_threshold_ = static_cast<float>(declare_parameter("dust_ratio_threshold", 0.2));
+    dust_count_threshold_ = static_cast<int>(declare_parameter("dust_count_threshold", 10));
+    dust_kernel_size_ = static_cast<int>(declare_parameter("dust_kernel_size", 2));
+    dust_buffering_frames_ = static_cast<int>(declare_parameter("dust_buffering_frames", 10));
+    dust_buffering_interval_ = static_cast<int>(declare_parameter("dust_buffering_interval", 1));
```

Comment:
@badai-nguyen Could you remove default parameters in declare_parameter.
We must define the params in config file or launch file

Reply:
@miursh I removed default paramters here https://github.com/autowarefoundation/autoware.universe/pull/3212/commits/e20652ccc66b0a31ed3d4dd6ca9ff8762f2298e3

Reply:
Please.... do not confuse the initial value of variables with default parameters.
If you remove the initial value of a variable, the algorithm will not work well, right?
Also, if you remove the default values of parameters from the code, the node will not start unless you give it to it from elsewhere.

Have you tested whether it launches correctly after your change?

Reply:
fixed https://github.com/autowarefoundation/autoware.universe/pull/3212/commits/93cee3c9b33f8f776f77fbd48ec5b420c54e8788

Reply:
@miursh I appologize for my mistake. :bow:  Thank you for your fixing.